qemu/tests/cfg/bridge_qinq.cfg

```diff
@@ -0,0 +1,29 @@
+- bridge_qinq:
+    only Linux
+    only bridge
+    type = bridge_qinq
+    private_bridge = tmpbr
+    copy_qinq_script = linux_qinq/qinq.sh
+    ping_count = 100
+    net_mask = "24"
+    ip_vm = "192.168.1.2"
+    vlan_id = 10
+    L1tag_iface = "v1v${vlan_id}"
+    L1tag_iface_ip = "192.168.10.10"
+    tcpdump_cmd = "tcpdump -xxvvleni %s > %s &"
```

Comment:
```suggestion
    tcpdump_cmd = "tcpdump -xxvvleni %s icmp > %s &"
```

Reply:
If add 'icmp' in the tcpdump cmd, line 288 in py file cannot get any output from tcpdump.

Reply:
Yes, my fault. `icmp` only only works for L1tag, `ether proto` can grep the ethertype

Reply:
Yes, my fault. `icmp` only only works for L1tag, `ether proto` can grep the ethertype